ci/prepare_build_environment.ps1

```diff
@@ -0,0 +1,3 @@
+Echo "install protocol buffer compiler..."
+choco install protoc
+choco install cuda --version=11.7.1.51694
```

Comment:
One more note, I installed cuda by downloading complete `.exe` file (3.05GB), and it installed to `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.3` by default.

Here I just use `choco` and assume it has similar behaviour, but be aware it may cause issue here.